mcp_clickhouse/mcp_env.py

```diff
@@ -29,12 +29,20 @@ class ClickHouseConfig:
         CLICKHOUSE_SEND_RECEIVE_TIMEOUT: Send/receive timeout in seconds (default: 300)
         CLICKHOUSE_DATABASE: Default database to use (default: None)
         CLICKHOUSE_PROXY_PATH: Path to be added to the host URL. For instance, for servers behind an HTTP proxy (default: None)
+        MCP_TRANSPORT: Transport method for mcp server, either "sse", "streamable-http", or "stdio" (default: "stdio")
```

Comment:
nit: can we call this `CLICKHOUSE_MCP_TRANSPORT` that way all the env's have the same prefix.